### PR TITLE
Feature/us11

### PR DESCRIPTION
**Pull request by jore-averbeck**

### CODE REVIEW

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🚀 . You only should add the `.idea` folder to your .gitignore.

File: components/Form/index.js [hunk 1]
```diff
@@ -65,86 +152,129 @@ export default function Form({ idea = {}, onSubmit }) {
...
           maxLength={150}
         />
-        <ButtonContainer>
-          <StyledButton>{router.query.id ? "save" : "add"}</StyledButton>
-        </ButtonContainer>
+        <StyledButtonContainer>
+          <StyledSaveButton>
+            <StyledButtonText>
+              {router.query.id ? "save" : "add"}{" "}
+            </StyledButtonText>
```
Comment (On-Fi) [inline components/Form/index.js:264]: {" "} no need for a empty comment 😜

File: pages/index.js [hunk 2]
```diff
@@ -70,63 +201,82 @@ export default function HomePage({ onToggleFavourites, favouriteIdeas }) {
...
+                <CardListItem key={suggestion.item._id}>
+                  <Card
+                    image={suggestion.item.image}
+                    title={suggestion.item.title}
+                    hashtags={suggestion.item.hashtags}
+                    onToggleFavourites={onToggleFavourites}
+                    favouriteIdeas={favouriteIdeas}
+                    id={suggestion.item._id}
+                  />
+                  {/* <StyledLink href={`/ideaDetails/${suggestion.item._id}`}>
```
Comment (On-Fi) [inline pages/index.js:232]: No need for this one as well :P

File: pages/index.js [hunk 3]
```diff
@@ -70,63 +201,82 @@ export default function HomePage({ onToggleFavourites, favouriteIdeas }) {
...
+                  <Card
+                    image={idea.image}
+                    title={idea.title}
+                    hashtags={idea.hashtags}
+                    onToggleFavourites={onToggleFavourites}
+                    favouriteIdeas={favouriteIdeas}
+                    id={idea._id}
+                    idea={idea}
+                  />
+                  {/* <StyledLink href={`/ideaDetails/${idea._id}`}>
```
Comment (On-Fi) [inline pages/index.js:250]: No need for this one as well :P

**Review by On-Fi — APPROVED** [review]
LGTM 👍🏼 
just a few comments to remove :)

**Pull request merged**